contributed/matmul.py

```diff
@@ -0,0 +1,86 @@
+from neuronxcc.nki import baremetal, benchmark
+import neuronxcc.nki.language as nl
+import neuronxcc.nki.isa as ni
+import numpy as np
+
+@benchmark(save_neff_name='file.neff', save_trace_name='profile.ntff', additional_compile_opt=' --disable-internal-io-dge ')
+def matmul(A_DRAM, B_DRAM, O_DRAM, K2, K1, K0, M2, M1, M0, N2, N1, N0):
```

Comment:
Could we document each of these parameters, its purpose and expected invariants?

Reply:
Is there a particular format you want the documentation in?

Reply:
python doc string for the function and its parameters is great, including relevant info about what semantic concept in the computation it maps to, dtype (if interesting), and shape/type

comments can just be normal python comments

Reply:
These should be added. Thanks!